tasks/libs/pipeline_notifications.py

```diff
@@ -7,6 +7,7 @@
 from .types import FailedJobType, Test
 
 DEFAULT_SLACK_CHANNEL = "#agent-platform"
+DEFAULT_JIRA_PROJECT = "AGENTR"
```

Comment:
```suggestion
DEFAULT_JIRA_PROJECT = "AGENT"
```

Typo? Or Jira withcraft?

Reply:
My bad it's the right one !